src/bin/server.rs

```diff
@@ -1,6 +1,18 @@
+use clap::Parser;
 use rustdis::{server, Error};
 
+const PORT: u16 = 6379;
+
+#[derive(Parser, Debug)]
+struct Args {
+    /// The port to listen on
```

Comment:
```suggestion

```

Reply:
I shouldn't have committed this. Since it's a doc comment, clap uses it for the help.